onnxruntime/contrib_ops/cpu/sparse/sparse_attention_base.h

```diff
@@ -184,7 +184,7 @@ class SparseAttentionBase {
         // Concatenate past_k + k -> present_k
         // TODO: avoid copying mutiple times for a group.
         k = ConcatStateChunkGQA(past_key, k, present_key, present_buff_chunk_length, past_buff_chunk_length,
-                                past_chunk_length, kv_input_chunk_length, is_prompt, past_present_share_buffer,
+                                is_prompt ? 0 : past_chunk_length, kv_input_chunk_length, past_present_share_buffer,
```

Comment:
what is the result to change the the SparseAttention?

Reply:
made this change to SparseAttention because I had to change the ConcatStateChunkGQA function which is shared between sparse and group query attention in cpu. This makes it match previous behavior